runtime/moonbase/src/weights/cumulus_pallet_parachain_system.rs

```diff
@@ -61,10 +61,10 @@ impl<T: frame_system::Config> cumulus_pallet_parachain_system::WeightInfo for We
 		// Proof Size summary in bytes:
 		//  Measured:  `82`
 		//  Estimated: `3517`
-		// Minimum execution time: 2_264_000 picoseconds.
-		Weight::from_parts(2_385_000, 3517)
-			// Standard Error: 22_805
-			.saturating_add(Weight::from_parts(293_496_971, 0).saturating_mul(n.into()))
+		// Minimum execution time: 2_128_000 picoseconds.
+		Weight::from_parts(91_853_295, 3517)
```

Comment:
Huge increase

Reply:
It's weird, I don't see a big difference in the final weights comparison: `393.97ms `to `393.71ms` using [subweight](https://github.com/ggwpez/substrate-weight-compare).